testsuite/scripts/gha_test_run_proto.json

```diff
@@ -176,13 +146,19 @@
             "suite":"TestUnitTest",
             "test":"test_setUpClass_already_happened",
             "skip":true,
-            "skipReason":"UnitTest fails (FIXME)"
+            "skipReason":"test fails when run the first time after interpreter boot"
         },
         {
             "suite":"TestWebView",
-            "test":"*",
+            "test":"test_findText_found",
+            "skip":true,
+            "skipReason":"test likely doesn't work inside the CI"
```

Comment:
Is this comment coming from your tests? We should be able to run GUI classes fine, as long as they are not IDE-only. IIRC my experience was that these tests _sometimes_ failed, so again something between the testing framework, CI environment and the test itself.

Reply:
Yeah, this is my comment. The 2 "findText" tests are commented in the source. It says they only work if the window is being displayed.

Reply:
But we can "display" windows in CI, can't we? We do run a virtual framebuffer device (on Linux); on macOS the graphic server is always available AFAIU. The test is using `Window` so not an IDE class.

Reply:
When I looked at the logs, one the "findText" tests was passing and the other not. When I looked at the source, it wasn't clear to me why either would fail. That's why I marked them both as skipped. We can work on this issue some other time.

Reply:
I agree that they should be skipped, but I was commenting on your change to the reason it's skipped - I thought the original comment was more accurate. Maybe not a big deal either way.

Reply:
Ok, yeah I agree. All of the comments here should be re-evaluated. I'll change it back when I squash the commits to clean up the history.